### PR TITLE
[Feature][Transforms-V2] LLM transforms Support custom field name

### PR DESCRIPTION
**Pull request by hawk9821**
### Purpose of this pull request
LLM transforms Support custom field name

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
E2E:  TestLLMIT#testLLMWithOpenAIOutputColumnName

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/transform-v2/llm.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ more.
 ## Options
 
 | name                   | type   | required | default value |
-| ---------------------- | ------ | -------- | ------------- |
+|------------------------| ------ | -------- | ------------- |
 | model_provider         | enum   | yes      |               |
 | output_data_type       | enum   | no       | String        |
+| output_column_name     | string | no       | String        |
```
Comment (Hisoka-X) [inline docs/en/transform-v2/llm.md]:
```suggestion
| output_column_name     | string | no       | llm_output        |
```

File: docs/zh/transform-v2/llm.md [hunk 2]
```diff
@@ -8,12 +8,13 @@
 
 ## 属性
 
-| 名称                   | 类型   | 是否必须 | 默认值 |
-| ---------------------- | ------ | -------- | ------ |
+| 名称                     | 类型   | 是否必须 | 默认值    |
+|------------------------| ------ | -------- |--------|
 | model_provider         | enum   | yes      |        |
 | output_data_type       | enum   | no       | String |
+| output_column_name     | string | no       |        |
```
Comment (Hisoka-X) [inline docs/zh/transform-v2/llm.md]:
```suggestion
| output_column_name     | string | no       |  llm_output      |
```

File: seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/llm/LLMTransformConfig.java [hunk 3]
```diff
@@ -37,6 +37,12 @@ public class LLMTransformConfig extends ModelTransformConfig {
                     .noDefaultValue()
                     .withDescription("The row projection field of each inference");
 
+    public static final Option<String> OUTPUT_COLUMN_NAME =
+            Options.key("output_column_name")
+                    .stringType()
+                    .noDefaultValue()
```
Comment (Hisoka-X) [inline seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/llm/LLMTransformConfig.java]:
```suggestion
                    .defaultValue("llm_output")
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @hawk9821 !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**